root/static/scripts/edit/URLCleanup.js

```diff
@@ -1565,6 +1565,7 @@ export const CLEANUPS: CleanupEntries = {
     clean(url) {
       url = url.replace(/^(?:https?:\/\/)?([^/]+\.)?beatport\.com\//, 'https://$1beatport.com/');
       url = url.replace(/^https:\/\/(?:(?:classic|pro|www)\.)?beatport\.com\//, 'https://www.beatport.com/');
+      url = url.replace(/^(https:\/\/www\.beatport\.com)\/[a-z]{2}(\/release\/)/i, '$1$2'); //remove country code from release urls
```

Comment:
Having a comment seems sensible but to keep line length in check I'd put it above the line rather than at the end. Also our style requires a space after `//` (see the eslint test failure)